rulebook.tex

```diff
@@ -1630,6 +1639,15 @@ \subsection{Task Fulfillment and Scoring}
 awarded. Performing a step for a later order which has not yet been
 announced cannot be awarded.
 
+For a competitive order, the same scoring scheme for in-time, delayed, and late
```

Comment:
What's the difference between delayed and late delivery?

Reply:
Delayed: within 10 seconds after the deadline, late: more than 10 seconds after the deadline (as specified in the scoring table)

Reply:
Ah, got ya.